plugins/scaffolder/src/next/TemplateWizardPage/TemplateWizardPage.tsx

```diff
@@ -73,6 +78,20 @@ export const TemplateWizardPage = (props: TemplateWizardPageProps) => {
     name: templateName,
   });
 
+  const { value: editUrl } = useAsync(async () => {
+    const data = await catalogApi.getEntityByRef(templateRef);
+    return data?.metadata.annotations?.['backstage.io/edit-url'] || '';
```

Comment:
I think that there's a constant from `@backstage/catalog-model` you can use here instead:

https://github.com/backstage/backstage/blob/703d305a4534a431f5441192e15647367234d1e8/packages/catalog-model/src/entity/constants.ts#L36

Reply:
I fixed it